R/bundle.R

```diff
@@ -833,17 +832,16 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters,
 }
 
 validatePackageSource <- function(pkg) {
-  msg <- NULL
```

Comment:
The old version of `validatePackageSource` was effectively a no-op, since the repository had an `NA` value rather than `NULL`, as identified during https://github.com/rstudio/rsconnect/pull/493.